libcxx/include/__tuple

```diff
@@ -254,29 +254,30 @@ template <class ..._Tp> struct __tuple_types {};
 
 namespace __indexer_detail {
 
-template <size_t _Idx, class _Tp>
-struct __indexed { using type _LIBCUDACXX_NODEBUG_TYPE = _Tp; };
+template <size_t _Idx, class ..._Types>
+struct _one_of;
```

Comment:
This would be better off being called `_nth_of`, right?

Reply:
Sure, I can rename it to that.